data/json/items/gunmod/sights.json

```diff
@@ -37,14 +37,14 @@
   {
     "id": "improve_sights",
     "type": "GUNMOD",
-    "name": "improved iron sights",
+    "name": "iron sights",
```

Comment:
Yeah, the 'improved' part of the name implies that they're a step up from what's already there. This makes it seem more like we're adding sights to a gun that has none.